src/index.js

```diff
@@ -1,19 +1,20 @@
 var wrapper = require('@pact-foundation/pact-node');
-var Promise = require("bluebird");
+var bluebird = require("bluebird");
 var deasync = require('deasync');
 
 var runPactMockServer = function (pacts, logger) {
+	var promise = Promise || bluebird;
```

Comment:
Move this up to the top with bluebird, should be `var promise = Promise || require('bluebird');`. This way, it only has to do the logic once on require.

Reply:
You're right! Done!